src/components/InventoryV2/DownloadReportModal.tsx

```diff
@@ -0,0 +1,20 @@
+import ExportCsvModal from 'src/components/common/ExportCsvModal';
+import NurseryInventoryService, { SearchInventoryParams } from 'src/services/NurseryInventoryService';
+
+interface DownloadReportModalProps {
```

Comment:
we have a mixed usage of `type` vs `interface`.. maybe we should stick to one?

Reply:
I think `type` is supposed to be used for more "transient" data types and `interface` is supposed to be used for "public facing" (or I guess exported and used in several places?) data types. This theory could be outdated! I agree we should agree on some convention... admittedly I think I just use `type` everywhere right now 😓